### PR TITLE
[TextFields] Improving hit zone for multiline

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: components/TextFields/src/MDCMultilineTextField.m [hunk 1]
```diff
@@ -400,6 +400,19 @@ - (CGFloat)estimatedTextViewLineHeight {
...
 
+#pragma mark - Touch (UIView)
+
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
+  UIView *receiver = [super hitTest:point withEvent:event];
+
+  // We do this so the hit zone to become first responder is larger than just the small text view.
+  if (receiver == self && self.isEditing == NO) {
+    return self.textView;
+  }
```
Comment (ianegordon) [inline components/TextFields/src/MDCMultilineTextField.m:411]: Does this have any impact on accessibility when VoiceOver mode is on?
Do we want to set a custom accessibility frame if so?

**Review by ianegordon — CHANGES_REQUESTED** [review]

File: components/TextFields/src/MDCMultilineTextField.m (hunk 1 above)
Comment (willlarche) [reply components/TextFields/src/MDCMultilineTextField.m:411]: This does not affect existing VO but VO isn't ideal on multiline. Should be improved in separate issue.

Comment (willlarche) [reply components/TextFields/src/MDCMultilineTextField.m:411]: #1895

**Review by ianegordon — APPROVED** [review]

**Comment by ianegordon** [comment]
LGTM.  Please merge when Travis is happy.

**Pull request merged**